master/buildbot/changes/gerritchangesource.py

```diff
@@ -300,12 +300,12 @@ def describe(self):
             project = "all projects"
 
         if self.branch is not None:
-           branch = "branch %s" % self.branch
+           branch = "branch '%s'" % self.branch
         else:
            branch = "all branches"
 
         if not self.process:
             status = "[NOT CONNECTED - check log]"
-        msg = ("GerritChangeSource watching the remote "
+        msg = ("GerritChangeSource watching project %s (%s) in remote "
```

Comment:
You are right, I will correct that. The reason why it's the way it is is because the project string can be "all projects" or some specific project.